newrelic-php-agent/values.yaml

```diff
@@ -1,11 +1,48 @@
-# Default values for simple-echo.
+# Default values for newrelic-php-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-image:
-  repository: chatwork/newrelic-php-agent
-  tag: latest
-  pullPolicy: IfNotPresent
-  command: []
-
-nameOverride: ""
-fullnameOverride: ""
\ No newline at end of file
+newrelic:
+  image:
+    repository: chatwork/newrelic-php-agent
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+    command: []
+
+  address: /tmp/.newrelic.sock
+
+  port: 31339
+
+  livenessProbe:
+    failureThreshold: 2
+    initialDelaySeconds: 10
+    periodSeconds: 60
+
+  readinessProbe:
+    failureThreshold: 2
+    periodSeconds: 5
+
+  daemonset:
+    enabled: true
+    extraPorts: []
+    extraEnv: []
+    extraEnvFrom: []
+    extraVolumes: []
+    extraVolumeMounts: []
+
+  deployment:
+    enabled: false
+    replicaCount: 1
+    extraPorts: []
+    extraEnv: []
+    extraEnvFrom: []
+    extraVolumes: []
+    extraVolumeMounts: []
+
+  ingress:
+    enabled: false
+
+  service:
+    type: ClusterIP
```

Comment:
@cw-niinuma 
`daemonset` does not require `service`.
Please change the `service` so that it can be selected with the `enabled` option.